lib/js/patterns/tabs.js

```diff
@@ -2,75 +2,111 @@
 import * as dom from '../helpers/dom';
 import * as classy from '../helpers/classy';
 import * as event from '../helpers/event';
+import Guid from 'guid';
+import bus from '../helpers/bus';
 
 // ┌──────┐
 // │ Tabs │
 // └──────┘
 // tabbed content pane
 function tabs () {
-  var tabs = dom.findElements('.js-tab');
-  var tabGroups = dom.findElements('.js-tab-group');
-  var tabSections = dom.findElements('.js-tab-section');
+  bus.on('tabs:bind', bindTabs);
+  bus.on('tabs:active', setTab);
 
-  // set max width for each tab
-  tabGroups.forEach(function (tab) {
-    tab.setAttribute('aria-live', 'polite');
-    tab.children[0].setAttribute('role', 'tablist');
-    var tabsInGroup = tab.querySelectorAll('.js-tab');
-    var percent = 100 / tabsInGroup.length;
-    for (var i = 0; i < tabsInGroup.length; i++) {
-      tabsInGroup[i].style.maxWidth = percent + '%';
-    }
-  });
+  function bindTabs () {
+    var tabs = dom.findElements('.js-tab');
+    var tabGroups = dom.findElements('.js-tab-group');
+    var tabSections = dom.findElements('.js-tab-section');
 
-  function switchTab (e) {
-    event.preventDefault(e);
+    // set max width for each tab
+    tabGroups.forEach(function (tab) {
+      tab.setAttribute('aria-live', 'polite');
+      groupId(tab);
+      tab.children[0].setAttribute('role', 'tablist');
+      var tabsInGroup = tab.querySelectorAll('.js-tab');
+      var percent = 100 / tabsInGroup.length;
+      for (var i = 0; i < tabsInGroup.length; i++) {
+        tabsInGroup[i].style.maxWidth = percent + '%';
+      }
+    });
 
-    var tab = dom.closest('js-tab', event.target(e));
-    var tabGroup = dom.closest('js-tab-group', tab);
-    var tabs = tabGroup.querySelectorAll('.js-tab');
-    var contents = tabGroup.querySelectorAll('.js-tab-section');
-    var index = dom.nodeListToArray(tabs).indexOf(tab);
+    tabs.forEach(function (tab) {
+      tab.setAttribute('aria-expanded', 'false');
+      tab.setAttribute('role', 'tab');
+      tab.setAttribute('tabindex', '0');
+      event.add(tab, event.click(), clickTab);
+      event.add(tab, 'keyup', enterTab);
+    });
 
-    classy.removeActive(tabs);
-    classy.removeActive(contents);
+    tabSections.forEach(function (section) {
+      section.setAttribute('role', 'tabpanel');
+      var isOpen = classy.has(section, 'is-active');
+      if (isOpen) {
+        section.setAttribute('aria-expanded', 'true');
+      } else {
+        section.setAttribute('aria-expanded', 'false');
```

Comment:
on line 73 you're passing `setAttribute` `false` and here you're using `"false"`. I'm not sure if that matters.